draft-ietf-privacypass-batched-tokens.md

```diff
@@ -75,10 +77,17 @@ ways:
 1. Improving server and client issuance efficiency by amortizing the cost of the
    VOPRF proof generation and verification, respectively.
 
-# Client-to-Issuer Request {#client-to-issuer-request}
+For the Publicly Verifiable Token issuance protocol, it allows for a single TokenRequest
+to be sent that encompasses multiple tokens. The cost remains linear.
+
+# Batched Privately Verifiable Token
+
+This section describes a batched issuance protocol for select token types, including 0x0001 (defined in {{RFC9578}}) and 0xF91A (defined in this document).
+
```

Comment:
We need to note that this variant is primarily meant to be more efficient, it comes at the cost of losing the flexibility to choose a different truncated_token_key_id for every token.